redash/query_runner/elasticsearch.py

```diff
@@ -132,6 +130,48 @@ def add_column_if_needed(mappings, column_name, friendly_name, result_columns, r
                     "type" : mappings.get(column_name, "string")})
                 result_columns_index[friendly_name] = result_columns[-1]
 
+        def getRow(rows, row):
```

Comment:
We're using [PEP8](https://www.python.org/dev/peps/pep-0008/) as a style guide. The TL;DR is that we use `snake_case` for names (~~`getRow`~~ -> `get_row`).